build.gradle

```diff
@@ -120,7 +120,7 @@ subprojects {
 
 ext {
     // NB: due to version.json generation by parsing this file, the next line must not have any if/then/else logic
-    neo4jVersion = "4.4.5"
+    neo4jVersion = "4.4.8"
```

Comment:
shouldn't this also be 4.4.9?

Reply:
We discussed this privately and as said java-driver and neo4j version do not normally have to match.